swap/src/main.rs

```diff
@@ -49,9 +50,11 @@ mod serde_peer_id;
 #[macro_use]
 extern crate prettytable;
 
+const MONERO_WATCH_ONLY_TEMP_WALLET_NAME: &str = "swap-tool-watch-only-tmp-wallet";
```

Comment:
The "temp" qualifier does not tell much about what this wallet is for.
Instead I'd suggest to name it "Watch Only Wallet" or "Blockchain Read Wallet" or "Watch Blockchain WAllet"

Reply:
I went for `swap-tool-blockchain-monitoring-wallet` because I feel it transports the meaning. Bit long but I don't mind about that :)